<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>fwbackups</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../fwbackups.qrc">
    <normaloff>:/application-icons/fwbackups-32.png</normaloff>:/application-icons/fwbackups-32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="overviewPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>24</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Welcome to fwbackups! Click on any of the buttons above to get started.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Status</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <property name="horizontalSpacing">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Total backup sets:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>&lt;i&gt;Not Loaded&lt;/i&gt;</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>&lt;i&gt;Not Loaded&lt;/i&gt;</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Failed backups since last check:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Operations in progress:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>&lt;i&gt;Not Loaded&lt;/i&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_1">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="backupSetsPage">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="topMargin">
             <number>12</number>
            </property>
            <item>
             <widget class="QPushButton" name="newSetButton">
              <property name="text">
               <string>New Set...</string>
              </property>
              <property name="icon">
               <iconset resource="../fwbackups.qrc">
                <normaloff>:/program-icons/program-icons/list-add.png</normaloff>:/program-icons/program-icons/list-add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editSetButton">
              <property name="text">
               <string>Edit Set...</string>
              </property>
              <property name="icon">
               <iconset resource="../fwbackups.qrc">
                <normaloff>:/program-icons/program-icons/preferences-system.png</normaloff>:/program-icons/program-icons/preferences-system.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteSetButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Delete Set</string>
              </property>
              <property name="icon">
               <iconset resource="../fwbackups.qrc">
                <normaloff>:/program-icons/program-icons/delete.png</normaloff>:/program-icons/program-icons/delete.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>12</height>
               </size>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="backupNowButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Backup Now</string>
              </property>
              <property name="icon">
               <iconset resource="../fwbackups.qrc">
                <normaloff>:/program-icons/program-icons/archive.png</normaloff>:/program-icons/program-icons/archive.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="restoreButton">
              <property name="text">
               <string>Restore</string>
              </property>
              <property name="icon">
               <iconset resource="../fwbackups.qrc">
                <normaloff>:/program-icons/program-icons/undo.png</normaloff>:/program-icons/program-icons/undo.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="burnButton">
              <property name="text">
               <string>Burn</string>
              </property>
              <property name="icon">
               <iconset resource="../fwbackups.qrc">
                <normaloff>:/program-icons/program-icons/media-optical.png</normaloff>:/program-icons/program-icons/media-optical.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QListView" name="listView"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="operationsPage">
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="horizontalSpacing">
           <number>12</number>
          </property>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <property name="spacing">
             <number>-1</number>
            </property>
            <property name="topMargin">
             <number>12</number>
            </property>
            <item>
             <widget class="QPushButton" name="viewDetailsButton">
              <property name="text">
               <string>View Details...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cancelOperationButton">
              <property name="text">
               <string>Cancel Operation</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QListView" name="listView_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="logViewerPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextEdit" name="logTextEdit">
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>-1</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>5</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="saveLogButton">
            <property name="text">
             <string>Save to File</string>
            </property>
            <property name="icon">
             <iconset resource="../fwbackups.qrc">
              <normaloff>:/program-icons/program-icons/document-save.png</normaloff>:/program-icons/program-icons/document-save.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearLogButton">
            <property name="text">
             <string>Clear Log</string>
            </property>
            <property name="icon">
             <iconset resource="../fwbackups.qrc">
              <normaloff>:/program-icons/program-icons/clear.png</normaloff>:/program-icons/program-icons/clear.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshLogButton">
            <property name="text">
             <string>Refresh</string>
            </property>
            <property name="icon">
             <iconset resource="../fwbackups.qrc">
              <normaloff>:/program-icons/program-icons/refresh.png</normaloff>:/program-icons/program-icons/refresh.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="restorePage">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Set"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Sets"/>
    <addaction name="actionExport_Sets"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionOverview"/>
    <addaction name="separator"/>
    <addaction name="actionBackup_Sets"/>
    <addaction name="actionOne_Time_Backup"/>
    <addaction name="actionRestore"/>
    <addaction name="separator"/>
    <addaction name="actionOperations"/>
    <addaction name="actionLog_Viewer"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionCheck_for_Updates"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="windowTitle">
    <string>fwbackups</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOverview_2"/>
   <addaction name="separator"/>
   <addaction name="actionBackup_Sets_2"/>
   <addaction name="actionOne_Time_Backup_2"/>
   <addaction name="actionRestore_2"/>
   <addaction name="separator"/>
   <addaction name="actionOperations_2"/>
   <addaction name="actionLog_Viewer_2"/>
  </widget>
  <action name="actionNew_Set">
   <property name="text">
    <string>New Set...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionImport_Sets">
   <property name="text">
    <string>Import Sets...</string>
   </property>
  </action>
  <action name="actionExport_Sets">
   <property name="text">
    <string>Export Sets...</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionEdit_Set">
   <property name="text">
    <string>Edit Set...</string>
   </property>
  </action>
  <action name="actionDuplicate_Set">
   <property name="text">
    <string>Duplicate Set</string>
   </property>
  </action>
  <action name="actionDelete_Set">
   <property name="text">
    <string>Delete Set</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionOverview">
   <property name="text">
    <string>Overview</string>
   </property>
  </action>
  <action name="actionBackup_Sets">
   <property name="text">
    <string>Backup Sets</string>
   </property>
  </action>
  <action name="actionOne_Time_Backup">
   <property name="text">
    <string>One-Time Backup...</string>
   </property>
  </action>
  <action name="actionLog_Viewer">
   <property name="text">
    <string>Log Viewer</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>Restore...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionOverview_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../fwbackups.qrc">
     <normaloff>:/program-icons/program-icons/home.png</normaloff>:/program-icons/program-icons/home.png</iconset>
   </property>
   <property name="text">
    <string>Overview</string>
   </property>
  </action>
  <action name="actionBackup_Sets_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../fwbackups.qrc">
     <normaloff>:/program-icons/program-icons/drive-harddisk.png</normaloff>:/program-icons/program-icons/drive-harddisk.png</iconset>
   </property>
   <property name="text">
    <string>Backup Sets</string>
   </property>
  </action>
  <action name="actionOne_Time_Backup_2">
   <property name="icon">
    <iconset resource="../fwbackups.qrc">
     <normaloff>:/program-icons/program-icons/drive-optical.png</normaloff>:/program-icons/program-icons/drive-optical.png</iconset>
   </property>
   <property name="text">
    <string>One-Time Backup</string>
   </property>
  </action>
  <action name="actionLog_Viewer_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../fwbackups.qrc">
     <normaloff>:/program-icons/program-icons/filetype-text.png</normaloff>:/program-icons/program-icons/filetype-text.png</iconset>
   </property>
   <property name="text">
    <string>Log Viewer</string>
   </property>
  </action>
  <action name="actionRestore_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../fwbackups.qrc">
     <normaloff>:/program-icons/program-icons/undo.png</normaloff>:/program-icons/program-icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
  </action>
  <action name="actionOperations_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../fwbackups.qrc">
     <normaloff>:/program-icons/program-icons/refresh.png</normaloff>:/program-icons/program-icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Operations</string>
   </property>
   <property name="toolTip">
    <string>Operations</string>
   </property>
  </action>
  <action name="actionOperations">
   <property name="text">
    <string>Operations</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../fwbackups.qrc"/>
 </resources>
 <connections/>
</ui>
